### DISCUSSION TITLE
handbook/enterprise_security/devsecops/toolchains/sast/

### DISCUSSION THREAD

**Discussion by giscus** [SEC-CAFE/handbook#6] [General]
# handbook/enterprise_security/devsecops/toolchains/sast/

静态应用安全测试(SAST)列表

https://sec.cafe/handbook/enterprise_security/devsecops/toolchains/sast/

**Comment by user1430** [comment]
1. 推荐使用多扫描工具集成平台进行扫描时间，将代码质量、规范、安全相关扫描能力集中在一次扫描中进行。减少运动式重复扫描治理
2. 能够减少研发参与的情况下，推荐直接越过研发落地SAST扫描能力，直接对仓库发起扫描和入库扫描
